유저 정보 수정

mkh4014 opened the pull request:
userId-user02로 데이터를 임의로 받아와 테스트한 상태이며 추후에 추가 작업이 필요함 
현재 user02의 데이터를 수정 가능, 취소 버튼까지 구현(취소를 클릭시 전 화면으로 돌아감)

mkh4014 committed:
유저 정보 수정

userId-user02로 데이터를 임의로 받아와 테스트한 상태이며 추후에 추가 작업이 필요함
현재 user02의 데이터를 수정 가능, 취소 버튼까지 구현

yyujjin commented on src/main/webapp/WEB-INF/views/user/updateSuccess.jsp:
회원 수정 완료 후에 페이지로 이동 돼서 회원정보 수정 성공 되었다고 유저에게 알려주는 방향보단 알림창을 띄워서 완료되었다고 하는 방향이 좋을것 같다고 제안드려봅니다!  해당 기능을 구현하려면 자바스크립트로 구현해야 합니당

yyujjin commented on src/main/java/com/jjobkorea/controller/UserController.java:
```diff
@@ -100,20 +100,35 @@ public HashMap<String, Boolean> checkId(@RequestParam("userId") String userId) {
+    	
+    	userService.updateUser(userDTO);
+    	log.info("업데이트 된 userDTO : {}",userDTO);
+    	session.setAttribute("user", userDTO);
```
지금은 세션에 유저 정보를 직접 저장하는 코드는 필요 없을 것 같습니다! 
125번 session 파라미터랑 129번줄은 삭제하는 방향이 좋을 것 같습니다!

mkh4014 committed:
유저 정보 수정

userId-user02로 데이터를 임의로 받아와 테스트한 상태이며 추후에 추가 작업이 필요함
현재 user02의 데이터를 수정 가능, 취소 버튼까지 구현(취소를 클릭시 전 화면으로 돌아감),
수정이 완료되면 다른 폼으로 이동하는데 나중에 자바스크립트 추가 작업으로 알림창 띄우기
